first draft of indiana mapping

guegueng opened the pull request:
This mapping is really simple, so I think this should be correct. The biggest area I’m unsure of is the first 5 lines

guegueng committed:
first draft of indiana mapping

anarchivist commented on heidrun/indiana_dc.rb:
```diff
@@ -0,0 +1,64 @@
+
+  sourceResource :class => DPLA::MAP::SourceResource do
+  
+  collection :class => DPLA::MAP::Collection, :each => record.field('xmlns:header', 'xmlns:setSpec'), :as => :coll do
```
1) The `:each` clause should be the following: `header.field('xmlns:setSpec')`
2) (minor) This line needs to be indented two more spaces, to line up with the `end` below it

anarchivist commented on heidrun/indiana_dc.rb:
```diff
@@ -0,0 +1,64 @@
+      title coll
+    end
+
+
```
(minor) Remove extra line

anarchivist commented on heidrun/indiana_dc.rb:
```diff
@@ -0,0 +1,64 @@
+
+    description record.field('dc:description')
+
+
```
(minor) Remove extra line

anarchivist commented on heidrun/indiana_dc.rb:
```diff
@@ -0,0 +1,64 @@
+      providedLabel place
+    end
+
+
```
(minor) Remove extra line

anarchivist commented on heidrun/indiana_dc.rb:
```diff
@@ -0,0 +1,64 @@
+
+
+    dcformat record.field('dc:format')
+
```
Do we need a mapping for `genre` here? From the spreadsheet:

> dc:type when the value matches 
> Book
> Film/Video
> Manuscript
> Maps
> Music
> Musical Score
> Newspapers
> Nonmusic
> Photograph/Pictorial Works
> Serial

This could also be something that could be dealt with in enrichment (mapping from any existing values for `dctype` to `genre`). I'm going to cc @no-reply because I think there's a philosophical angle he might want us to consider.

guegueng replied:
I think we might have talked about this and the solution was to map the type property in all of the fields and run enrichment to remove unecessary values. But I'll wait for @no-reply as well.

anarchivist commented on heidrun/indiana_dc.rb:
```diff
@@ -0,0 +1,64 @@
+    description record.field('dc:description')
+
+
+    dcformat record.field('dc:format')
```
Do we also need to pull in values from `dc:type`? From the spreadsheet:

> dc:format; non-DCMIType values from dc:type

Compare with my [comment below](https://github.com/dpla/heidrun-mappings/pull/6/files#r26264048) about `genre`. cc @no-reply

guegueng replied:
to include two properties do I do:
dcformat record.field('dc:format', 'dc:type')
?

anarchivist replied:
It would be like this: `dcformat record.fields('dc:format', 'dc:type')` - note that it's **`fields`** and not **`field`**

anarchivist commented on heidrun/indiana_dc.rb:
```diff
@@ -0,0 +1,64 @@
+  provider :class => DPLA::MAP::Agent do
+    uri 'http://dp.la/api/contributor/indiana'
+    label 'Indiana Memory'
+    #I'm guessing here as to the naming of the above things
```
:ok: I'd confirm with @rudeamy re: the `label`.

anarchivist commented:
Great work, @guegueng. Most or all of these changes are minor, except for the change related to `collection`.

rudeamy commented:
Nice! Indiana Memory is the name they are using according to the ingest
form:
https://docs.google.com/a/dp.la/document/d/11bUrkXL_KLRkGJiH4YQ3nOq97DYmMBapCXyaWSQWdt0/edit

A

On Wed, Mar 11, 2015 at 5:53 PM, Mx A. Matienzo <notifications@github.com>
wrote:

> Great work, @guegueng <https://github.com/guegueng>. Most or all of these
> changes are minor, except for the change related to collection.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dpla/heidrun-mappings/pull/6#issuecomment-78390398>.
>



-- 
Amy Rudersdorf
Assistant Director for Content
Digital Public Library of America

amy@dp.la
(608) 501-1502

guegueng commented on heidrun/indiana_dc.rb:
```diff
@@ -0,0 +1,64 @@
+
+    dctype record.field('dc:type')
+  end
+end
```
looking at this, I'm wondering if both of the "end" are necessary? I'm not sure what the first one is ending?

anarchivist replied:
Yes, both are necessary. The `end` on line 63 closes the `sourceResource` block that starts on line 25, and the one on line 64 closes the `Krikri::Mapper.define` block at the beginning of the file.

guegueng committed:
edits to indiana mapping

all of the edits suggested by @anarchivist including adding the mapping
for genre (which was also directed to @no-reply) because in retrospect
I remember this conversation and I think this is right.

guegueng commented:
Ready for testing.

guegueng committed:
update to indiana mapping

update to collection mapping for indiana mapping, also updated indenting

moltude commented:
After talking w/@guegueng I'm closing this PR and deleting the branch b/c the mapping is out of date with IN datasource and needs to be rewritten.
